pages/api/interview.ts

```diff
@@ -0,0 +1,69 @@
+import { ChatGPTMessage } from "@/constant/constant";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<any>
+) {
+  try {
+    if (req.method === "POST") {
+      const newMsg: any = await chatGPTHandler(req);
+      return res.status(200).json({
+        data: newMsg.choices[0].message,
+      });
+    }
+  } catch (error) {
+    return res.status(401).json({ error });
+  }
+}
+
+const chatGPTHandler = async (req: NextApiRequest): Promise<Response> => {
+  const body = await JSON.parse(req.body);
+
+  const requestHeaders: Record<string, string> = {
+    "Content-Type": "application/json",
+    Authorization: `Bearer ${process.env.OPENAI_API_KEY ?? "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"}`,
+  };
+
+  const { skills, yearsOfExperience, jobRole, jobDescription } = body.userInfo;
+
+  const messages: ChatGPTMessage[] = [
+    {
+      role: "system",
+      content: `You are a friendly technical interviewer with name TechTalQ, 
```

Comment:
Can we move this system text to another config file?
I suggest to make a file, chat-init.config.json.
Define the initial messages in that file for the system and the user. Use those messages and append the conversation in them via code